esxi/resource_guest.go

```diff
@@ -42,6 +42,13 @@ func resourceGUEST() *schema.Resource {
 				Default:     nil,
 				Description: "Local path to source ovf files.",
 			},
+			"uefi": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
Not true...  You can modify this on the fly without destroying your VM.

Reply:
you can , but i think the VM won't start if you installed in bios mode

Reply:
It is indeed possible to reinstall the OS, so I turned off ForceNew.